commands/list.go

```diff
@@ -16,29 +16,33 @@
 package commands
 
 import (
+	"bytes"
 	"fmt"
+	"io"
 	"net"
 	"strconv"
 	"strings"
+	"text/tabwriter"
 
 	"github.com/AlekSi/pointer"
 	"github.com/percona/pmm/api/inventorypb/json/client"
 	"github.com/percona/pmm/api/inventorypb/json/client/agents"
 	"github.com/percona/pmm/api/inventorypb/json/client/services"
 	"github.com/percona/pmm/api/inventorypb/types"
+	"github.com/sirupsen/logrus"
 	"gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/percona/pmm-admin/agentlocal"
 )
 
 var listResultT = ParseTemplate(`
-Service type                Service name                        Address and port       Service ID
+Service type{{"\t"}}Service name{{"\t"}}Address and port{{"\t"}}Service ID
 {{ range .Services }}
-{{- printf "%-27s" .HumanReadableServiceType }} {{ printf "%-35s" .ServiceName }} {{ printf "%-22s" .AddressPort }} {{ .ServiceID }}
```

Comment:
since we are using tabs, we can remove this `printf "%-27s"` and other `printf` usages in this template. So could you remove them, please?

Reply:
Yeap, I will remove them.

Reply:
Thanks, @JiriCtvrtka already did that :)

Reply:
Great then :)